ankihub/gui/js_message_handling.py

```diff
@@ -119,11 +119,8 @@ def _on_js_message(handled: Tuple[bool, Any], message: str, context: Any) -> Any
         ah_nids = kwargs.get("noteIds")
         note_suspension_states = _get_note_suspension_states(ah_nids)
 
-        from .reviewer import reviewer_sidebar
-
         _post_message_to_ankihub_js(
-            message={"noteSuspensionStates": note_suspension_states},
-            web=reviewer_sidebar.content_webview if reviewer_sidebar else context.web,
```

Comment:
After the `reviewer_sidebar` get set up, which happens when you open the reviewer, this would always send the message to the `reviewer_sidebar.content_webview`.
So the smart search dialog wouldn't get the message.